doc/man3/CMS_add0_cert.pod

```diff
@@ -48,8 +51,7 @@ As the I<0> implies CMS_add0_cert() adds I<cert> internally to I<cms> and it
 must not be freed up after the call as opposed to CMS_add1_cert() where I<cert>
 must be freed up.
 
-The same certificate or CRL must not be added to the same cms structure more
-than once.
+The same certificate must not be added to the same cms structure more than once.
```

Comment:
Why the change here?

Reply:
This change fixes the documentation.
The code does not implement a uniqueness restriction on CRLs.